client/src/app/site/pages/meetings/modules/poll/components/entitled-users-table/entitled-users-table.component.html

```diff
@@ -18,14 +18,36 @@
                         </div>
 
                         <!-- Delegation -->
-                        <div *ngIf="entry.vote_delegated_to_user_id">
+                        <div *ngIf="entry.vote_delegated_to_user_id && !entry.delegation_user_merged_into_id">
                             <span>
                                 ({{ 'represented by' | translate }} {{ entry.vote_delegated_to.getShortName().trim() }})
                             </span>
                         </div>
+                        <div *ngIf="entry.vote_delegated_to_user_id && entry.delegation_user_merged_into_id">
+                            <span>
+                                ({{ 'represented by old account' | translate }}) {{ delegatedMergedIntoName(entry) }}
```

Comment:
should probably be `represented by old account of ...`

Reply:
I have changed this.